components/RegistrationForm/RegistrationForm.js

```diff
@@ -28,14 +28,8 @@ const registrationSchema = Yup.object().shape({
   'confirm-password': Yup.string()
     .required(validationErrorMessages.required)
     .oneOf([Yup.ref('password')], validationErrorMessages.passwordMatch),
-  firstName: Yup.string()
-    .required(validationErrorMessages.required)
-    // eslint-disable-next-line no-useless-escape
-    .matches(/^[a-zA-Zà-žÀ-Ž \-]+$/, validationErrorMessages.name),
-  lastName: Yup.string()
-    .required(validationErrorMessages.required)
-    // eslint-disable-next-line no-useless-escape
-    .matches(/^[a-zA-Zà-žÀ-Ž \-]+$/, validationErrorMessages.name),
+  firstName: Yup.string(),
+  lastName: Yup.string(),
```

Comment:
We still want these listed as required! Also, you'll want to remove some code in test files that assert `validationErrorMessages.name` is rendered.

Reply:
I marked these fields as required. Also, I was not able to find any tests that assert that `validationErrorMessages.name` is rendered when the input fields do not match the regex given